### ISSUE TITLE
Case insensitive redirects

### ISSUE THREAD

**Issue by chmchm** [johngodley/redirection#53]
Hello, love the ease of the plugin.
I would request adding an option for case insensitivity to the source URL. As I went thru the support at wordpress.org I realized you had to hard code all of the options.
Thanks!

**Comment by davidimh** [comment]
In the meantime you can use regular expressions to accomplish this in one pass.  I have a page at /boston and I wanted any capitalization of that to go there (i.e. "Boston" or "BOSTON").

My source URL is "[B][Oo][Ss][Tt][Oo][Nn]".  Note that the lowercase "b" is not included.  Note also that I am a complete novice at regular expressions and may be overlooking something!

**Comment by lleds** [comment]
Thanks David that worked. 
For long term I can also really use a checkbox to ignore case of the source url.
Thanks!

**Comment by davidimh** [comment]
Glad it was helpful.  I forgot I had posted this or I probably would have updated it.  I added the lowercase "b" into the first pairing because, without it, if someone were to type "bOSTON" (i.e. caps-lock reversed) it wouldn't have been caught by the original expression. So, the source URL now is "[Bb][Oo][Ss][Tt][Oo][Nn]".

I agree that the ability to have case insensitivity would be a great addition to an already fantastic plug-in.
